alanEappen/assignments/fastSlow/lc141/lc141.js

```diff
@@ -0,0 +1,78 @@
+// Problem Statement #
+
+//Given the head of a Singly LinkedList, write a function to determine if the LinkedList has a cycle in it or not.
+//Given:
+//head
+//Goal: to return a boolean to see if the linkedlist has a cycle.
+//variables:
+//slowPtr, fastPtr is equal to the head
+//fastPtr and fastPtr.next exists proceed with the loop
+//slowPtr.next() 
+//fastPtr.next().next()
+//if(slowPtr == fastPtr)
+//return true
+//if not a cycle 
+//return false
+class Node {
+  constructor(value, next = null) {
+    this.value = value;
+    this.next = next;
+  }
+}
+
+const has_cycle = function (head) {
+  let slowPtr = head;
+  let fastPtr = head;
+
+  while (fastPtr && fastPtr.next) {
+
+    slowPtr = slowPtr.next
+    fastPtr = fastPtr.next.next
+    if (slowPtr === fastPtr) {
+      return true
+    }
+  }
+
+  return false
+}
```

Comment:
I'm loving the code clarity!~ Good use of naming variables. Where is the love for typing it twice??!?!? #KeepUpGoodWork #TypeItTwice

Make sure to type the solution out once more and resubmit at your earliest convenience!